.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and verify PR Title
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  verify-pr-title:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: semantic-pull-request
+        id: pr-title
+        uses: amannn/action-semantic-pull-request@v5.5.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: build application
+        if: steps.pr-title.outcome == 'success'
+        run: yarn build
```

Comment:
This isn't needed since this gh action will only be responsible for checking the semantic pr title, the `build` is already taken by vercel so it isn't needed here.